server/deployster_service.go

```diff
@@ -37,13 +38,16 @@ func NewDeploysterService(listen string, version string, username string, passwo
 
 func (ds *DeploysterService) ConfigureRoutes() {
 	fleetClient := fleet.NewClient("/var/run/fleet.sock")
+	dockerClient, _ := docker.NewClient("unix:///var/run/docker.sock")
```

Comment:
Yeah, I agree.  It hasn't been a huge priority yet since pretty much the only deploy target for this is CoreOS which these are the defaults for (and this runs in Docker anyway, so you can mount other volumes into this container at these paths easily).

But I agree, we should get these pulled out at some point.